### DISCUSSION TITLE
How to set start date in search_tweets?

### DISCUSSION THREAD

**Discussion by VinceGa** [tweepy/tweepy#1720] [Questions] [answered]
I want to set the search language and start + end dates in the search API. It seems i can only use Search_tweets(). But this API does not support start/end dates. Any thoughts on what API I should use?

**Comment by Harmon758** [comment, marked as answer]
[`API.search_tweets`](https://tweepy.readthedocs.io/en/v4.4.0/api.html#tweepy.API.search_tweets) uses the [standard search API](https://developer.twitter.com/en/docs/twitter-api/v1/tweets/search/api-reference/get-search-tweets), with which you can use the `since` and `until` [standard search operators](https://developer.twitter.com/en/docs/twitter-api/v1/tweets/search/guides/standard-operators) to specify start and end dates. For language, you can use the `lang` parameter of the method/endpoint.

**Reply by VinceGa** [reply]
Here's what I did:
 for tweet in tweepy.Cursor(api.search_tweets, tag, lang='en',since= '202111120000').items(100):

and the result is: "Unexpected parameter: since"

Any hint?

**Reply by Harmon758** [reply]
As I said and as shown in the linked documentation, `since` is an operator you can use within the query itself, with a date in ISO 8601 format. It is not a parameter for the standard search API endpoint that `API.search_tweets` uses.
